src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReportTest.java

```diff
@@ -174,13 +174,13 @@ public void testConsiderCaseIfValidHearingStatusReportHdr() {
         ListingData reportListingData = report.processHearingsByHearingTypeRequest(listingDetails, submitEvents);
 
         AdhocReportType adhocReportType = reportListingData.getLocalReportsSummaryHdr();
-        assertEquals(6, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getTotal())?0:Integer.parseInt(adhocReportType.getTotal()));
-        assertEquals(1, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getHearing())?0:Integer.parseInt(adhocReportType.getHearing()));
-        assertEquals(1, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getHearingCM())?0:Integer.parseInt(adhocReportType.getHearingCM()));
-        assertEquals(1, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getCosts())?0:Integer.parseInt(adhocReportType.getCosts()));
-        assertEquals(1, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getHearingPrelim())?0:Integer.parseInt(adhocReportType.getHearingPrelim()));
-        assertEquals(1, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getReconsider())?0:Integer.parseInt(adhocReportType.getReconsider()));
-        assertEquals(1, adhocReportType == null || Strings.isNullOrEmpty(adhocReportType.getRemedy())?0:Integer.parseInt(adhocReportType.getRemedy()));
+        assertEquals(6, Strings.isNullOrEmpty(adhocReportType.getTotal())?0:Integer.parseInt(adhocReportType.getTotal()));
```

Comment:
what's the purpose of converting the getTotal() value to an int at all? assert equals can be done on the string?